src/ol_infrastructure/applications/bootcamps_ecommerce/__main__.py

```diff
@@ -0,0 +1,203 @@
+                        "s3:GetObject*",
+                        "s3:DeleteObject*",
+                    ],
+                    "Resource": [
+                        f"arn:aws:s3:::{bootcamps_storage_bucket_name}",
+                        f"arn:aws:s3:::{bootcamps_storage_bucket_name}/*",
+                    ],
+                },
+                {
+                    "Effect": "Allow",
+                    "Action": ["execute-api:Invoke", "execute-api:ManageConnections"],
+                    "Resource": "arn:aws:execute-api:*:*:*",
+                },
+            ],
+        },
+        stringify=True,
+        parliament_config={
+            "PERMISSIONS_MANAGEMENT_ACTIONS": {
+                "ignore_locations": [{"actions": ["s3:putobjectacl"]}]
+            },
+            "RESOURCE_EFFECTIVELY_STAR": {},
+        },
+    ),
+)
+
+bootcamps_vault_backend_role = vault.aws.SecretBackendRole(
+    f"bootcamps-app-{stack_info.env_suffix}",
+    name=f"bootcamps-app-{stack_info.env_suffix}",
+    backend="aws-mitx",
+    credential_type="iam_user",
+    policy_arns=[bootcamps_iam_policy.arn],
+)
+
+# Create RDS instance
+bootcamps_db_security_group = ec2.SecurityGroup(
+    f"bootcamps-db-access-{stack_info.env_suffix}",
+    description=f"Access control for the bootcamps DB in {stack_info.name}",
+    ingress=[
+        ec2.SecurityGroupIngressArgs(
+            protocol="tcp",
+            from_port=DEFAULT_POSTGRES_PORT,
+            to_port=DEFAULT_POSTGRES_PORT,
+            cidr_blocks=["0.0.0.0/0"],
+            ipv6_cidr_blocks=["::/0"],
+            description="Allow access over the public internet from Heroku",
+        ),
+        ec2.SecurityGroupIngressArgs(
+            protocol="tcp",
+            from_port=DEFAULT_POSTGRES_PORT,
+            to_port=DEFAULT_POSTGRES_PORT,
+            security_groups=[data_vpc["security_groups"]["integrator"]],
+        ),
+    ],
+    egress=[
+        ec2.SecurityGroupEgressArgs(
+            from_port=0,
+            to_port=0,
+            protocol="-1",
+            cidr_blocks=["0.0.0.0/0"],
+            ipv6_cidr_blocks=["::/0"],
+        )
+    ],
+    tags=aws_config.merged_tags(
+        {"name": "bootcamps-db-access-applications-{stack_info.env_suffix}"}
+    ),
+    vpc_id=apps_vpc["id"],
+)
+
+bootcamps_db_config = OLPostgresDBConfig(
+    instance_name=f"bootcamps-db-applications-{stack_info.env_suffix}",
+    password=bootcamps_config.require("db_password"),
+    subnet_group_name=apps_vpc["rds_subnet"],
+    security_groups=[bootcamps_db_security_group],
+    tags=aws_config.tags,
+    db_name="bootcamps",
+    public_access=True,
+    **defaults(stack_info)["rds"],
+)
+bootcamps_db = OLAmazonDB(bootcamps_db_config)
+
+bootcamps_vault_backend_config = OLVaultPostgresDatabaseConfig(
+    db_name=bootcamps_db_config.db_name,
+    mount_point=f"{bootcamps_db_config.engine}-bootcamps-applications-{stack_info.env_suffix}",  # noqa: E501
```

Comment:
Because we have a different Vault cluster per env stage it's actually preferred to leave the env info out of the mount point. This should just be `f"{bootcamps_db_config.engine}-bootcamps"`